cli.ts

```diff
@@ -66,6 +68,23 @@ async function cli(args: ParsedArgs) {
     validateStatus: () => true,
   })
 
+  if ("action_attempt" in response.data) {
+    const { poll_for_action_attempt } = await prompts({
+      name: "poll_for_action_attempt",
+      message: "Would you like to poll the action attempt until it's ready?",
+      type: "toggle",
+      initial: true,
+      active: "yes",
+      inactive: "no",
+    })
+
+    if (poll_for_action_attempt) {
+      response.data.action_attempt = await pollActionAttemptUntilReady(
+        response.data.action_attempt.action_attempt_id
```

Comment:
I was wondering what should we return in cases when `action_attempt` is returned along with some other recourse like `access_code`. The issue is that if we return a resolved `action_attempt` then it will have an updated version of `access_code` in its `result` (if the action was successful) but the initial `access_code` which is created immediately on request will be outdated. So, it will be like 
```
{
  action_attempt: {
    result: {
      access_code: { // updated one }
    }
  }, 
  access_code: { // initial, outdated one }
}
```